### PR TITLE
Fixed lidar orientation and points frame

### PR DESCRIPTION
**Pull request by pijaro**
Fixed:
- lidar beam directions
- results in local frame

### CODE REVIEW

File: Code/Source/Lidar/LidarTemplateUtils.h [hunk 1]
```diff
@@ -17,6 +17,7 @@ namespace ROS2
     public:
         static LidarTemplate GetTemplate(LidarTemplate::LidarModel model);
         static size_t TotalPointCount(const LidarTemplate& t);
-        static AZStd::vector<AZ::Vector3> PopulateRayDirections(LidarTemplate::LidarModel model);
+        static AZStd::vector<AZ::Vector3> PopulateRayDirections(LidarTemplate::LidarModel model,
```
Comment (adamdbrw) [inline Code/Source/Lidar/LidarTemplateUtils.h:20]: could use a comment that angles are euler, radians

File: Code/Source/Lidar/LidarTemplateUtils.cpp [hunk 2]
```diff
@@ -74,8 +75,8 @@ namespace ROS2
             for (int layer = 0; layer < lidarTemplate.m_layers; layer++)
             {
                 // roll is equal to 0, so it's skipped in the calculations
```
Comment (adamdbrw) [inline Code/Source/Lidar/LidarTemplateUtils.cpp:77]: we should not leave out roll transformation

Comment (pijaro) [reply Code/Source/Lidar/LidarTemplateUtils.cpp:77]: Agreed, but not the scope of this PR. Most probably we will move towards quaternions anyway.

File: Code/Source/Lidar/LidarTemplateUtils.h (hunk 1 above)
Comment (pijaro) [reply Code/Source/Lidar/LidarTemplateUtils.h:21]: added comment

**Pull request merged**